### ISSUE TITLE
NVDA doesn't announce form elements and doesn't evoke action after one time pop-up is evoked.

### ISSUE THREAD

**Issue by PetrashkoEV** [nvaccess/nvda#6393]
NVDA doesn't announce button's/link's name and doesn't evoke action after one time pop-up is evoked. 

I created an example that simulates a pop-up opening. Please, take a look to the following example, where the issue fully reproducible.
http://codepen.io/PetrashkoEV/pen/BLzRjE?editors=1010

**Steps to reproduce:**

1.

**Comment by PetrashkoEV** [comment]
Sorry guys 
I by mistake pressed the Submit button.  :(
